content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding Windows worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Windows worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* A running [Windows Server 2019](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing)
+(or higher) instance with administrative access.
+* If you are using VXLAN/Overlay networking you must have also have
+[KB4489899](https://support.microsoft.com/help/4489899) installed.
+* A running kubeadm cluster created by `kubeadm init` and following the steps
+in the document [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+* `curl.exe` is installed and present in the `PATH` environment variable.
+
+<!-- steps -->
+
+## Adding Windows worker nodes
+
+{{< note >}}
+To facilitate the addition of Windows worker nodes to a cluster, PowerShell scripts from the repository
+https://sigs.k8s.io/sig-windows-tools are used.
+{{< /note >}}
+
+Do the following for each machine:
+
+1. Connect to the machine by using SSH or another method.
+1. Make sure you are Administrator or a privileged user.
+1. Open a PowerShell session.
+
+Then proceed with the steps outlined below.
+
+### Install containerd
+
+Run the following commands to install containerd:
+
+  ```PowerShell
+  curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/Install-Containerd.ps1
+  .\Install-Containerd.ps1 -ContainerDVersion 1.7.1
```

Comment:
```suggestion
  .\Install-Containerd.ps1 -ContainerDVersion  1.7.22
```

Reply:
should we point to the containerd runtime docs here instead? https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd

Reply:
i did not notice that there are windows install steps in https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd (inside the getting started link) that's great because we can just add a prereq for a container runtime.

ideally we should break the follwing into linux/windows eventually too:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl

and maybe have PrepareNode.ps be the install step for windows in that guide.

Reply:
the biggest differences between those docs are what we have are

- Checking and enabling the Hyper-V feature - which I don't think we care about for the purposes of this document 
- Setting the CNI bin and CNI config dir's in the containerd config - I think we do want this

I think if we can get the containerd docs updated to create and set the cni bin/config dirs in the containerd setup script we can move away from the containerd setup doc in sig-windows-tools.
@jsturtevant - WDYT?

Reply:
agree, That info should live in the containerd install doc

Reply:
i though about unrolling the install-containerd script but think i'm going to leave this task to SIG Windows. my main problem here is that i have no way to test the containerd install and CNI setup, ATM.

also, it seems out of scope of the PR.